### ISSUE TITLE
Question: Tag latest not up to date with last version tag 0.16.0 on Docker Hub

### ISSUE THREAD

**Issue by woicha** [powerman/dockerize#224]
Hello,

first of all thank you all for your work. We are using `dockerize` for the upcoming release of our examination server in an educational environment.

This issue is opened only because I have a question:
Is there a particular reason why the tag [`latest`](https://hub.docker.com/layers/dockerize/powerman/dockerize/latest/images/sha256-430a8f9642a92557e1dc185c6a0a5dd73175738b3fa174e3b40581efc536ab2d?context=explore) on [hub.docker.com](https://hub.docker.com/r/powerman/dockerize/tags) points to another and two months older image then the last released version tag [0.16.0](https://hub.docker.com/layers/dockerize/powerman/dockerize/0.16.0/images/sha256-8d74ffb49520ac75fd56dc3282ba473b087cc45141103cd4056bca09a357d26a?context=explore)?
In other words:
Could you please shortly explain your release and tagging policy, especially regarding the `latest` tag?

Yours sincerely
&nbsp;woicha [Thomas Walcher]

**Comment by powerman** [comment]
Thanks for drawing my attention to this issue.

It's not about a "policy", it's probably just some issue with CI/CD pipeline, I'll take a look and fix it.

Personally I prefer to use concrete version and thus I missed this issue with latest tag. This tag is supposed to point to build from latest commit in master branch, which may be newer than latest released version but must not be older than it.

As a quickfix I've manually updated latest to match 0.16.0.

**Comment by woicha** [comment]
Thank you for your answer and your fix.

Can the `latest` build from the master branch be considered stable or is it better for a productive environment to rely on a release like [0.16.0](https://hub.docker.com/layers/dockerize/powerman/dockerize/0.16.0/images/sha256-8d74ffb49520ac75fd56dc3282ba473b087cc45141103cd4056bca09a357d26a?context=explore)?

**Comment by powerman** [comment]
Master branch is stable. Any important feature or a fix will be released, so changes in master which wasn't released are usually some chore (dependency/docs/ci updates etc.) or minor code cleanup.

But it's recommended (in general, this isn't specific to dockerize) to always use released version - newer versions may have incompatible changes or slight but surprising behaviour changes, so it's recommended to always test version you use in production and avoid getting surprising updates because of "latest" tag.

**Comment by woicha** [comment]
Thanks again for your quick answer, the general advice and most of all for maintaining this repo.

I'll close the issue with this reply. Feel free to open it again, if you consider it necessary.
